wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -125,6 +129,18 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
         walletApplication.analyticsService
     }
 
+    // TODO: do we need only Platform types in here?
+    private val keyChainTypes = EnumSet.of(
+        AuthenticationKeyChain.KeyChainType.BLOCKCHAIN_IDENTITY,
+        AuthenticationKeyChain.KeyChainType.BLOCKCHAIN_IDENTITY_FUNDING,
+        AuthenticationKeyChain.KeyChainType.BLOCKCHAIN_IDENTITY_TOPUP,
+        AuthenticationKeyChain.KeyChainType.INVITATION_FUNDING,
+        AuthenticationKeyChain.KeyChainType.MASTERNODE_PLATFORM_OPERATOR,
+        AuthenticationKeyChain.KeyChainType.MASTERNODE_OPERATOR,
+        AuthenticationKeyChain.KeyChainType.MASTERNODE_VOTING,
+        AuthenticationKeyChain.KeyChainType.MASTERNODE_OWNER
+    )
```

Comment:
We only need platform types here.  The masternode types will be added when users go to the **Masternode Keys** page if those key types were not yet added to the wallet.